react-accordion-component/nimmys-app/src/App.js

```diff
@@ -0,0 +1,56 @@
+import { useState } from 'react';
+import './App.css';
+
+function Container() {
```

Comment:
Good, it works. However, it's not reusable. It will _only_ work for the specific data you typed in below.

Make it a reusable component that is data-driven. Pass the data as a prop.